src/figuresClasses.ts

```diff
@@ -1,19 +1,103 @@
 export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
 
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle'
 }
 
 export class Triangle implements Figure {
+  shape: Shape;
```

Comment:
```suggestion
  shape:  Figure['shape'] = 'triangle';
```
do this for all inherits